src/main/resources/application.conf

```diff
@@ -2,6 +2,8 @@ etc-client {
 
   client-id = "etc-client"
 
+  keys-file = "nodeId.keys"
```

Comment:
Could you add some docs explaining syntax of a `keys-file`? This is not a blocker.